src/olympia/addons/tests/test_models.py

```diff
@@ -3178,6 +3178,19 @@ def test_pending_queue(self):
         addons = Addon.objects.get_pending_review_queue(admin_reviewer=True)
         assert set(addons) == set(expected_addons)
 
+    def test_human_review_queue_entries_have_no_due_date(self):
+        no_due_date = addon_factory(version_kw={'needs_human_review': True})
```

Comment:
If you add an assertion I think you'll find that it does have a due date - because the `needs_human_review` forces it. You'd have to explicitly pass `'due_date': None` for that not to happen. I'm surprised the test is passing...

Reply:
Your branch is missing https://github.com/mozilla/addons-server/commit/d960b4e3a7e0044e8d8559691ef84019dc3dc107 which has the logic to set `due_date` if `needs_human_review` is also set, which is why your test is passing despite not doing what I suggest. I recommend updating it with `master` and fixing the test before merging, just in case.

Reply:
Is that because of https://github.com/mozilla/addons-server/blob/60fdf84e050974524443979145ca6ea20a0ea362/src/olympia/versions/models.py#L1081-L1090? I am happy to update the test, the fact that it is passing without it is concerning? How can we make sure we're testing the right thing?

Reply:
Yep, that was it, and adding an explicit `None` due date makes it pass. Thanks!